debian/pat-configure.1

```diff
@@ -0,0 +1,17 @@
+.TH PAT 1 "2017-09-04" "Linux" "Pat Configure"
```

Comment:
Disclaimer: I have no idea about best practices regarding man pages.

Why does this mention Linux? My hope is that we can include these man pages for most platforms (including Linux, macOS and *BSD).

What do you think Andrew?

Reply:
Most of the man page formatting guides I read have "Linux" there just to show what platform the page is written for. If you want this to be cross platform I can just remove that with no problem.

Reply:
It is the footer line right?

For `man git`, `man ls` and `man perl` they all show the project's name and version.

I don't think we should include the version in every man page, but maybe replace it with `Pat`? I want to emphasize that I really have no clue about this, so please do it at you see fit.